src/EditorFeatures/CSharpTest/CodeActions/MoveType/MoveTypeTests.MoveToNewFile.cs

```diff
@@ -1275,6 +1275,99 @@ await TestMoveTypeToNewFileAsync(
             code, codeAfterMove, expectedDocumentName, destinationDocumentText);
     }
 
+    [WpfFact, WorkItem("https://github.com/dotnet/roslyn/issues/19613")]
+    public async Task MoveTypeWithDirectives3()
+    {
+        var code =
+            """
+            public class Goo
+            {
+                #region Region
+                public class [||]Bar
+                {
+                }
+                #endregion
+            }
+            """;
+        var codeAfterMove =
+            """
+            public partial class Goo
+            {
+
+                #region Region
+                #endregion
+            }
+            """;
+
+        var expectedDocumentName = "Bar.cs";
+        var destinationDocumentText =
+            """
+            public partial class Goo
+            {
+                #region Region
```

Comment:
Just to make sure I'm reading this correctly, the directive is expected to be in both places?

Reply:
it's a stretch to say 'expected'.  But at the very least we want it to be *correct*.